src/Form/FormControl.js

```diff
@@ -11,6 +11,7 @@ export const styleSheet = createStyleSheet('FormControl', () => {
       display: 'flex',
       flexDirection: 'column',
       position: 'relative',
+      transform: 'translateZ(0)', // #6076 prevents animation glitches of children to bubble up
```

Comment:
From what I can see of the video, it could be Chrome disabling the anti-aliasing to keep 60fps.